components/cam/src/physics/cam/micro_p3.F90

```diff
@@ -177,26 +177,36 @@ SUBROUTINE p3_init_a(lookup_file_dir,version_p3)
     !PMC: deleted ice-ice collision lookup table here b/c only used for nCat>1.
     ! So there is no need to fill lookup values for lookup table 2.
 
+   return
+
   END SUBROUTINE p3_init_a
 
   subroutine p3_get_tables(mu_r_user, revap_user, vn_user, vm_user)
     ! This can be called instead of p3_init_b.
+    implicit none
```

Comment:
Not that it matters, but `implicit none` at module level covers all contained subroutines.